rsa.go

```diff
@@ -75,3 +76,81 @@ func VerifyRecoverRSASignature(publicKey, signature []byte) ([]byte, error) {
 	recoveredBytes := C.GoBytes(unsafe.Pointer(rout), C.int(routlen))
 	return recoveredBytes, nil
 }
+
+// VerifyRSASignature verifies that a signature is valid for some data and a Public Key
+// - Parameter publicKey: The OpenSSL EVP_PKEY public key in DER format
+// - Parameter signature: The signature to verify in DER format
+// - Parameter data: The data used to generate the signature
+// - Parameter digest: The name of the digest to use. The currently supported values are: sha1, sha224, sha256, sha384, sha512, ripemd160, rsassapss
```

Comment:
The `rsassapss` I feel like doesn't fit here.  Maybe we should have a 5th config parameter or 2 additional parameter s for `rsa_padding_mode` and `rsa_pss_saltlen` settings?

Reply:
I moved the mapping out and made a map for the additional options.

Reply:
`rsassapss` is not a digest type, it's a type of signature algorithm.  I don't think `GetDigestByName` would work with `rsassapss`, does it?

Reply:
No, but it gets mapped to sha256 (based on the NFC sample code) which is the value that gets passed into this function.

Reply:
I removed `rsassapss` from the comment which was a relic from the previous commit.